packages/core/src/base/staticClient.ts

```diff
@@ -13,9 +13,9 @@ export { DefaultCache, Response };
 import { Resource } from "./resource";
 import { BaseClient } from "./client";
 import { sdkLogger } from "./sdkLogger";
+import fetchToCurl from "fetch-to-curl";
```

Comment:
Are global imports or local imports supposed to go first?

Reply:
don't think it matters. I will move next to other global import to keep it consistent